R/importPSIMITAB.R

```diff
@@ -0,0 +1,107 @@
+library(stringr)
+
```

Comment:
@RicardoRam - I am encouraging everyone to pull-request early, so I/we can comment directly into code. FWIW they can poaste R CMD check into the Issue when the code is really ready to by merged.